src/components/otp-exercise/otp.js

```diff
@@ -1,86 +1,86 @@
-// import React, {useEffect, useState} from 'react';
-// import {TextInput, View, StyleSheet, StatusBar, Alert} from 'react-native';
+import React, {useEffect, useState} from 'react';
+import {TextInput, View, StyleSheet, StatusBar, Alert} from 'react-native';
 
-// const inputs = new Array(6).fill(0);
+const inputs = new Array(6).fill(0);
 
-// function OTP() {
-//   const [otpTextInput] = useState([]);
-//   const [otp, setOtp] = useState(Array(6).fill(''));
+function OTP() {
+  const [otpTextInput] = useState([]);
+  const [otp, setOtp] = useState(Array(6).fill(''));
 
-//   useEffect(() => {
-//     otpTextInput[0].focus();
-//   }, []);
+  useEffect(() => {
```

Comment:
There is an autofocus prop in text field. Lets see if that works.

Reply:
done